.orchestra/config/components/toolchain/lib/gdb.lib.yml

```diff
@@ -42,22 +43,47 @@ configure: |
     --disable-gold \
     --disable-gprof \
     --disable-ld \
-    --disable-gdbserver \
+    --enable-gdbserver \
     --enable-64-bit-bfd \
     --disable-install-libbfd \
     --disable-install-libiberty \
     --without-guile \
     --with-system-readline \
     --enable-nls \
-    --enable-tui=no \
+    --enable-tui \
     --with-mpfr="$ORCHESTRA_ROOT" \
     --without-included-gettext \
     --disable-shared \
     --enable-obsolete \
     --without-stage1-ldflags \
+    --with-python="$ORCHESTRA_ROOT/bin/python" \
+    --with-system-gdbinit="$ORCHESTRA_ROOT/(@= triple @)/etc/gdb/gdbinit" \
     CFLAGS="-w -ggdb3 -O3" \
     CXXFLAGS="-w -ggdb3 -O3"
-build_system: make
+install: |
+  cd "$BUILD_DIR"
+  (@= make @)
+  (@= make @) install DESTDIR="$DESTDIR"
+
+  GDBINIT="$DESTDIR$ORCHESTRA_ROOT/(@= triple @)/etc/gdb/gdbinit"
+  mkdir -p "$(dirname "$GDBINIT")"
+
+  cat >"$GDBINIT" <<'EOF'
+  set pagination off
+  EOF
+
+  (@- if triple.startswith("x86_64"): @)
```

Comment:
There's also `host`. `x86_64` is a cross toolchain.
But I think the `.gdbinit` thing is non-portable and we should just drop it.

Reply:
No, `host`'s `triple` variable starts with `x86_64` and I specifically looked for something to specifically recognize all x86_64 targets.